dom-events/buttons.js

```diff
@@ -0,0 +1,28 @@
+function handleClick(event) {
+  console.log('button clicked');
+  console.log('  event:', event);
+  console.log('  event.target:', event.target);
+}
+
+var $clickButton = document.querySelector('.click-button');
+
+$clickButton.addEventListener('click', handleClick);
+
+function handleMouseover(event) {
+  console.log('button hovered');
+  console.log('  event:', event);
+  console.log('  event.target:', event.target);
+}
+
+var $hoverButton = document.querySelector('.hover-button');
+
+$hoverButton.addEventListener('mouseover', handleMouseover);
+
+function handleDoubleClick(event) {
+  console.log('button double-clicked');
+  console.log('  event:', event);
+  console.log('  event.target:', event.target);
+}
+var doubleClickButton = document.querySelector('.double-click-button');
+
+doubleClickButton.addEventListener('dbclick', handleDoubleClick);
```

Comment:
The event you're listening on here is misspelled, should be `dblclick`
If you test it right now as-is, it doesn't work